test/work_item_repository.go

```diff
@@ -32,6 +32,17 @@ type WorkItemRepository struct {
 		result1 *app.WorkItem
 		result2 error
 	}
+	ReorderStub        func(ctx context.Context, before string, wi app.WorkItem) (*app.WorkItem, error)
+	reorderMutex       sync.RWMutex
+	reorderArgsForCall []struct {
+		ctx    context.Context
+		before string
+		wi     app.WorkItem
+	}
+	reorderReturns struct {
+		result1 *app.WorkItem
+		result2 error
```

Comment:
@DhritiShikhar does it make sense to rename `result1` to `wi` and and `result2` to `err`?